src/include/storage/tuple_access_strategy.h

```diff
@@ -122,7 +133,14 @@ class TupleAccessStrategy {
    */
   void InitializeRawBlock(RawBlock *raw, layout_version_t layout_version) const;
 
-  /* Vectorized Access */
+  /**
+   * @param slot tuple slot value to check
+   * @return whether the given slot is occupied by a tuple
+   */
+  bool ValidSlot(TupleSlot slot) const {
```

Comment:
Naming isn't super important and we can debate this, but I prefer something like Allocated or Present, especially if we're still calling Allocate() to get a slot in the first place. Valid implies to me there's some extra meaning or verification going on.